doc/source/ray-core/handling-dependencies.rst

```diff
@@ -62,6 +62,8 @@ It is installed dynamically on the cluster at runtime and cached for future use
 
 Runtime environments can be used on top of the prepared environment from :ref:`the Ray Cluster launcher <using-the-cluster-launcher>` if it was used.
 For example, you can use the Cluster launcher to install a base set of packages, and then use runtime environments to install additional packages.
+In contrast with the base cluster environment, a runtime environment will only be active for Ray processes.  (For example, if you use SSH to connect to a Ray node and try to import
+a ``pip`` package from a runtime environment, it will not be available.)
```

Comment:
"import" as in from a regular Python process. Can we make this clear, maybe even give an example? When and how does a runtime lib exactly become available? Only ever through the cluster launcher process? etc.

Reply:
@maxpumperla I've edited it to make it more concrete with an example. 

As for when it becomes available, it's addressed in the section directly below (not visible in the diff), is it clear enough? We could move more detail into the first few lines, but we also need to keep the first few lines as a brief overview.